### PR TITLE
PIM-5752: add option label in behat

### PR DESCRIPTION
**Pull request by juliensnz**
**Description**

Add the ability to set options label in behats

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | NA
| Added Behats                      | NA
| Changelog updated                 | NA
| Review and 2 GTM                  | Y
| Micro Demo to the PO (Story only) | NA
| Migration script                  | N
| Tech Doc                          | N

### CODE REVIEW

**Comment by skeleton** [comment]
GTM

File: features/Context/Page/Attribute/Creation.php [hunk 1]
```diff
@@ -64,6 +64,10 @@ public function addOption($name)
         $row  = end($rows);
 
         $row->find('css', '.attribute_option_code')->setValue($name);
+
+        foreach ($labels as $locale => $label) {
+            $row->find('css', sprintf('.attribute-option-value[data-locale="%s"]', $locale))->setValue($name, $label);
```
Comment (phaseinducer) [inline features/Context/Page/Attribute/Creation.php]: Did you test it ? Because it should not work with this first `$name` param.

**Comment by solivier** [comment]
gtm

File: features/Context/Page/Attribute/Creation.php (hunk 1 above)
Comment (juliensnz) [inline features/Context/Page/Attribute/Creation.php]: fixed :)

**Pull request merged**